cmd/loadTests.go

```diff
@@ -203,6 +217,7 @@ func init() {
 	rootCmd.Flags().BoolVar(&failFast, "fail-fast", false, "if you want the test to fail fast at first failure")
 	rootCmd.Flags().BoolVar(&disableMetrics, "disable-metrics", false, "if you want to disable metrics gathering")
 	rootCmd.Flags().IntVarP(&threadCount, "threads", "t", 1, "number of concurrent threads to execute")
+	rootCmd.Flags().BoolVarP(&randomString, "randomstring", "r", false, "are we to add random string to the user prefix?")
```

Comment:
```suggestion
	rootCmd.Flags().BoolVarP(&randomString, "randomstring", "r", false, "if you want to add random string to the user prefix")
```
@naftalysh please be consistent with the other help messages for boolean values

Reply:
Updated